include/gridtools/common/array.hpp

```diff
@@ -197,3 +184,14 @@ namespace gridtools {
     /** @} */
 
 } // namespace gridtools
+
+namespace std {
+    template <class T, size_t D>
```

Comment:
Did we never use `gridtools::tuple_size` etc.?

Reply:
I see, we did but the abstraction is via tuple_util.

Reply:
There was one usage of it. I have replaced it to `tuple_util` stuff.